artifactory/cli.go

```diff
@@ -2311,9 +2325,50 @@ func transferConfigCmd(c *cli.Context) error {
 
 	// Run transfer config command
 	transferConfigCmd := transferconfig.NewTransferConfigCommand(sourceServerDetails, targetServerDetails).SetForce(c.Bool(cliutils.Force))
+	const patternSeparator = ";"
```

Comment:
The below code that reads the include / exclude repositories is duplicated for the `transfer-config` and `transfer-files` commands. Let's reuse it.